gitlab_runner/metadata.csv

```diff
@@ -0,0 +1,44 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+ci_docker_machines_provider_machine_creation_duration_seconds_bucket,histogram,,request,second,Histogram of Docker machine creation time,0,gitlab_runner,docker machine creation time
+ci_docker_machines_provider_machine_creation_duration_seconds_sum,gauge,,request,second,Sum of Docker machine creation time,0,gitlab_runner,sum docker machine creation time
+ci_docker_machines_provider_machine_creation_duration_seconds_count,gauge,,request,second,Count of Docker machine creation time,0,gitlab_runner,count docker machine creation time
+ci_docker_machines_provider_machine_states,gauge,,request,second,The current number of machines per state in this provider,0,gitlab_runner,total docker machines per state
+ci_runner_errors,counter,,request,second,The number of catched errors,0,gitlab_runner,errors
+ci_runner_version_info,gauge,request,,A metric with a constant '1' value labeled by different build stats fields,0,gitlab_runner,version info
```

Comment:
a column is missing, I think an empty one after `gauge`

Reply:
Great catch! Thanks